### ISSUE TITLE
[PATCH] Add helper stubMessagesControllerComponents for testing form controllers

### ISSUE THREAD

**Issue by hsn10** [playframework/playframework#8706]
Add new helper for testing controllers using forms more easily - stubMessagesControllerComponents

My implementation:

~~~scala
    def stubMessagesControllerComponents() : MessagesControllerComponents = {
       val stub = stubControllerComponents()
       new DefaultMessagesControllerComponents(
           new DefaultMessagesActionBuilderImpl(stubBodyParser(AnyContentAsEmpty),stub.messagesApi)(stub.executionContext),
           DefaultActionBuilder(stub.actionBuilder.parser)(stub.executionContext), stub.parsers, stub.messagesApi, stub.langs, stub.fileMimeTypes,
           stub.executionContext
           )
    }
~~~

**Comment by wsargent** [comment]
I think probably submit this as a PR?  Should be in https://github.com/playframework/playframework/blob/master/framework/src/play-test/src/main/scala/play/api/test/Helpers.scala#L589

**Comment by mkurz** [comment]
@hsn10 A Pull Request: https://github.com/playframework/playframework/compare
You fork the Play Framework git repository, commit your changes into a new branch and push that branch to your fork so you can open a pull request with that changes.
See

- https://help.github.com/articles/about-pull-requests/
- https://help.github.com/articles/creating-a-pull-request/

**Comment by wsargent** [comment]
Also see https://www.playframework.com/contributing

**Comment by wsargent** [comment]
Resolved by #8711
